mslib/msui/autoplot_dockwidget.py

```diff
@@ -346,7 +351,12 @@ def update_treewidget(self, parent, parent2, config_settings, treewidget, flight
             parent.refresh_signal_emit.emit()
 
         if treewidget.objectName() == "autoplotSecsTreeWidget":
-            if url is None:
+            if url == "":
+                QMessageBox.information(
+                    self,
+                    "WARNING",
+                    "Please select the URL,layer,styles and level(row information first)"
```

Comment:
Please select the URL, layer, styles and level (row information first)"

Reply:
done